store/tikv/coprocessor.go

```diff
@@ -646,25 +751,18 @@ func (it *copIterator) Next(ctx context.Context) (kv.ResultSubset, error) {
 			return nil, nil
 		}
 	} else {
-		for {
-			if it.curr >= len(it.tasks) {
```

Comment:
We should keep `it.curr`, since `copTask.respChan` may have multiple responses if a region split happens. It should not switch to next task until the responses of current task are consumed.

Reply:
How can we keep it? `it.tasks` is not a slice any more (it's a channel now).

Reply:
Done